src/js/components/uploadFabsFile/UploadFabsFileValidation.jsx

```diff
@@ -104,8 +104,16 @@ class UploadFabsFileValidation extends React.Component {
                     fabs_meta: response.fabs_meta
                 });
             })
-            .catch((err) => {
-                console.error(err);
+            .catch((error) => {
+                // Invalid Submission ID
+                if (error.httpStatus === 400) {
```

Comment:
The error object here returns "status" instead of "httpStatus".  Additionally, it would be good to handle all exceptions that aren't 400 errors with an "else" condition ( You might have to get some catch-all verbiage for this)